conf/buildpack.conf

```diff
@@ -41,6 +41,6 @@ webp_version="${webp_version:-1.3.2}" # Can be found here: https://storage.googl
 # Mandatory for multibytes strings starting with PHP 7.4
 libonig_version="${libonig_version:-6.9.8}"
 # From https://zlib.net/
-zlib_version="${zlib_version:-1.2.13}"
+zlib_version="${zlib_version:-1.3}"
```

Comment:
Ar you sure `1.3` works? And not `1.3.0`?

Reply:
Yep, it's very weird, but this is how they versioned it :|

See http://zlib.net/zlib-1.3.tar.gz VS http://zlib.net/zlib-1.3.0.tar.gz